Tests/Bricks/ImageBrickTests.swift

```diff
@@ -51,6 +51,11 @@ class ImageBrickTests: XCTestCase {
         imageURL = NSURL(fileURLWithPath: NSBundle(forClass: self.classForCoder).pathForResource("image0", ofType: "png")!)
     }
 
+    override func tearDown() {
+        super.tearDown()
+        BrickCollectionView.imageDownloader = NSURLSessionImageDownloader() // Reset the image downloader back to the default one (if a test had overwritten this)
```

Comment:
`BrickCollectionView.imageDownloader` is indeed static. The reason is that an app should have 1 type of way to download images. By making this static, every BrickCollectionView can benefit from this image downloader and is not reliable for injecting it.